docs/gettingstarted/index.rst

```diff
@@ -81,11 +81,8 @@ Getting started with Flocker
 
          Install the Flocker client on your Mac (requires Homebrew):
 
-         .. version-code-block:: console
-
-            you@laptop:~$ brew update && \
-              brew tap clusterhq/flocker && \
-              brew install flocker-|latest-installable|
+         .. task:: test_homebrew flocker-|latest-installable|
+            :prompt: you@laptop:~$
```

Comment:
I guess we're happy that the order of `brew tap ...` and `brew update` is changing (it's the other way around in `task_test_homebrew` and we're happy to have the use run the `brew test ...` command as well?  Considering that's what we're testing I think it seems like a great change, just want to be sure.